examples/with-tailwind/packages/ui/package.json

```diff
@@ -4,7 +4,8 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "exports": {
-    ".": "./dist"
+    ".": "./dist",
+    "./styles.css": "./dist/styles.css"
```

Comment:
Thanks for this, I have a few other changes I was staging up for today and this was in there.

Reply:
Okay, nice. I guess we can close this then?